python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -18,7 +18,9 @@
 
 # ==============================================================================
 
+
```

Comment:
Thanks for the changes to make style consistent! However, in the future can you make these changes as their own PR? It makes it a bit hard to find new changes that are not stylistic

Reply:
Sure, I will do that.